ci-scripts/update-ccv2.sh

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/env bash
+set -xe
+
+B2C_STORE="spartacusstore"
+B2B_STORE="b2bspastore"
+CCV2_B2C_STOREFRONT_PATH="$GHT_REPO/js-storefront/$B2C_STORE"
+CCV2_B2B_STOREFRONT_PATH="$GHT_REPO/js-storefront/$B2B_STORE"
+GH_BASE_URL="https://$GHT_USER:$GHT_PRIVATE_REPO_TOKEN@github.tools.sap/cx-commerce/$GHT_REPO.git"
+APP_MODULE_PATH="projects/storefrontapp/src/app/app.module.ts"
+B2C_CONFIG_PATH="projects/storefrontapp/src/app/spartacus/spartacus-b2c-configuration.module.ts"
+B2B_CONFIG_PATH="projects/storefrontapp/src/app/spartacus/spartacus-b2b-configuration.module.ts"
+SERVER_CONFIG_PATH="projects/storefrontapp/server.ts"
+
+function verify_branch_exist {
+    IS_BRANCH=$(git ls-remote --heads "$1" "$2")
+
+    if [ -z "$IS_BRANCH" ]; then
+        echo "Error. Can't find the branch $2. Verify the branch name exist"
+        exit 1
+    fi
+}
+
+function remove_pwa_config {
+    sed -i '/pwa:[[:blank:]]*{/,/^[[:space:]]*}/d' "$1"
+
+    cat "$1"
+
+    if grep -Fq "addToHomeScreen: true" "$1"
+    then
+        echo "PWA config has NOT been removed"
+        exit 1
+    else
+        echo "PWA config has SUCCESSFULLY been removed"
+    fi
+}
```

Comment:
Overall, it would be nice if at some point in the future, we had a way to provide a deployment specific config without having to use `sed` to modify the default files.  Just food for thought, but no action point required in this PR.